exec/jdbc/src/test/java/org/apache/drill/jdbc/test/TestJdbcQuery.java

```diff
@@ -122,6 +122,7 @@ public void testLikeNotLike() throws Exception{
       );
   }
 
+  @Ignore("Returns results in different order depeding on forkCount")
```

Comment:
Is this a regression due to this patch? Other wise, open a ticket for this issue.